### PR TITLE
Add CI tags to tracked spans

### PR DESCRIPTION
**Pull request by jirikuncar**
⬆️ 

uses https://github.com/DataDog/dd-trace-py/pull/1741

### CODE REVIEW

**Review by nmuesch — DISMISSED** [review]

File: .azure-pipelines/all.yml [hunk 1]
```diff
@@ -55,11 +59,12 @@ jobs:
         DD_AGENT_HOST: datadog-agent
         DD_ENV: prod
         DD_SERVICE: datadog-api-client-python
-        DD_TAGS: "team:integration-tools-and-libraries,build.requested_by:$(Build.RequestedForId),build.attempt:$(System.StageAttempt),pull_request.number:$(System.PullRequest.PullRequestNumber)"
+        DD_TAGS: "team:integration-tools-and-libraries"
         DD_TEST_CLIENT_API_KEY: $(datadogAPIKey)
         DD_TEST_CLIENT_APP_KEY: $(datadogAPPKey)
         DD_TRACE_ANALYTICS_ENABLED: "true"
         RECORD: "none"
+        PYTEST_ADDOPTS: "--ddtrace"
```
Comment (jirikuncar) [inline .azure-pipelines/all.yml:67]:
```suggestion
        PYTEST_ADDOPTS: "--ddtrace"
        DD_PYTEST_OPERATION_NAME: "test"
```

**Review by nmuesch — APPROVED** [review]
Changes look good 👍 
Though the CI seems to be failing

**Pull request merged**